setup.py

```diff
@@ -22,7 +22,7 @@ def parse_requirements(path: str):
 
 setuptools.setup(
     name="conbench",
-    version="1.62.0",
+    version="1.63.0",
```

Comment:
It's a provocative question -- but what does this even mean? :)

Reply:
Not much at all. In this case, I wanted to ensure that Arrow benchmarks were using a version of the `conbench` CLI that included this change. We'll probably get more meaning out of versioning in the future once we're uploading tools to PyPI.